### PR TITLE
Key links

### PR DESCRIPTION
**Pull request by FriederikeHanssen**

### CODE REVIEW

**Comment by FriederikeHanssen** [comment]
@nf-core-bot fix linting please

File: src/content/events/2024/hackathon-march-2024/index.mdx [hunk 1]
```diff
@@ -46,6 +46,13 @@ Please watch the introduction video below to get started:
     </div>
 </div>
 
+:::info{title="Key links"}
+
+-   [Gather Town]() _Note: You need the email you signed up with_
+-   [Project Board](https://github.com/orgs/nf-core/projects/65)
+-   [Wrap up slides](https://docs.google.com/presentation/d/10B2tb1si8WsRe1T2IfxgqvltuEaGZ_ItGQVXxiPh3IA/edit?usp=sharing)
+    :::
```
Comment (mashehu) [inline src/content/events/2024/hackathon-march-2024/index.mdx:54]:
```suggestion
:::
```

File: src/content/events/2024/hackathon-march-2024/index.mdx [hunk 2]
```diff
@@ -46,6 +46,13 @@ Please watch the introduction video below to get started:
     </div>
 </div>
 
+:::info{title="Key links"}
+
+-   [Gather Town]() _Note: You need the email you signed up with_
+-   [Project Board](https://github.com/orgs/nf-core/projects/65)
+-   [Wrap up slides](https://docs.google.com/presentation/d/10B2tb1si8WsRe1T2IfxgqvltuEaGZ_ItGQVXxiPh3IA/edit?usp=sharing)
```
Comment (mashehu) [inline src/content/events/2024/hackathon-march-2024/index.mdx:53]:
```suggestion
-   [Wrap up slides](https://docs.google.com/presentation/d/10B2tb1si8WsRe1T2IfxgqvltuEaGZ_ItGQVXxiPh3IA/edit?usp=sharing)

```

File: src/content/events/2024/hackathon-march-2024/index.mdx [hunk 3]
```diff
@@ -46,6 +46,14 @@ Please watch the introduction video below to get started:
     </div>
 </div>
 
+:::info{title="Key links"}
+
+-   [Gather Town]() _Note: You need the email you signed up with_
```
Comment (ewels) [inline src/content/events/2024/hackathon-march-2024/index.mdx:51]:
```suggestion
-   [Gather Town](https://app.gather.town/app/br78S294XBCL487A/nf-core-hackathon) _Note: You need the email you signed up with_
```

File: src/content/events/2024/hackathon-march-2024/index.mdx [hunk 4]
```diff
@@ -46,6 +46,14 @@ Please watch the introduction video below to get started:
     </div>
 </div>
 
+:::info{title="Key links"}
+
+-   [Gather Town](https://app.gather.town/app/br78S294XBCL487A/nf-core-hackathon) _Note: You need the email you signed up with_
+-   [Project Board](https://github.com/orgs/nf-core/projects/65)
```
Comment (ewels) [inline src/content/events/2024/hackathon-march-2024/index.mdx:52]:
```suggestion
-   [GitHub Hackathon Project Board](https://github.com/orgs/nf-core/projects/65)
```

File: src/content/events/2024/hackathon-march-2024/index.mdx [hunk 5]
```diff
@@ -46,6 +46,14 @@ Please watch the introduction video below to get started:
     </div>
 </div>
 
+:::info{title="Key links"}
+
+-   [Gather Town](https://app.gather.town/app/br78S294XBCL487A/nf-core-hackathon) _Note: You need the email you signed up with_
```
Comment (ewels) [inline src/content/events/2024/hackathon-march-2024/index.mdx:51]:
```suggestion
-   [Gather Town](https://app.gather.town/app/br78S294XBCL487A/nf-core-hackathon) _(Use the email you signed up with)_
```

File: src/content/events/2024/hackathon-march-2024/index.mdx [hunk 6]
```diff
@@ -24,6 +24,8 @@ Join us online **March 18-20, 2024**, for the nf-core hackathon! 🗓️
 The nf-core hackathon is free and is being held online (we will be back in [Gather](https://gather.town/)).
 Some community members are also hosting local sites.
 
+<!--
```
Comment (ewels) [inline src/content/events/2024/hackathon-march-2024/index.mdx:27]:
```suggestion
{/*
```

File: src/content/events/2024/hackathon-march-2024/index.mdx [hunk 7]
```diff
@@ -33,6 +35,8 @@ Some community members are also hosting local sites.
 -   Registration to attend the hackathon online closes (**March 18, 2024**)
 -   Hackathon (**March 18-20, 2024**)
 
+-->
```
Comment (ewels) [inline src/content/events/2024/hackathon-march-2024/index.mdx:38]:
```suggestion
*/}
```

**Comment by ewels** [comment]
@nf-core-bot fix linting

**Review by ewels — APPROVED** [review]
Thank you!

**Pull request merged**